geoapps/edge_detection/application.py

```diff
@@ -240,21 +282,23 @@ def update_name(self, _):
 
     def compute_trigger(self, _):
         param_dict = self.get_param_dict()
-        param_dict["geoh5"] = self.workspace
-
         if param_dict.get("objects", None) is None:
             return
 
-        with fetch_active_workspace(self.workspace):
-            new_params = EdgeDetectionParams(**param_dict)
+        with fetch_active_workspace(self.workspace, mode="r+") as ws:
```

Comment:
We will need to change the curve-apps.get_edges to avoid having to write to the file.

https://mirageoscience.atlassian.net/browse/GEOPY-1527